aten/src/ATen/native/TensorShape.cpp

```diff
@@ -203,9 +203,40 @@ Tensor& set_storage_cpu_(Tensor& result, Storage storage, int64_t storage_offset
   return result;
 }
 
-Tensor& set_(Tensor& result, const Tensor& storage, int64_t storage_offset, IntArrayRef size, IntArrayRef stride) {
+Tensor& set_storage_meta__symint(Tensor& result, Storage storage, c10::SymInt storage_offset, c10::SymIntArrayRef size, c10::SymIntArrayRef stride) {
```

Comment:
can these refs be const?

Reply:
no, they cannot. This is a long standing missing staircase that cannot be fixed without doing a very large and annoying c++ codemod